mage_ai/data_cleaner/cleaning_rules/shared.py

```diff
@@ -0,0 +1,7 @@
+COLUMN_NAME_QUOTE_CHARS = '+=-*&^%$! ?~|<>(){}[],.'
```

Comment:
It'll be better to move shared methods/files to this folder: https://github.com/mage-ai/mage-ai/tree/master/mage_ai/data_cleaner/shared